src/cgnslib.c

```diff
@@ -8868,10 +8869,10 @@ int cg_boco_write(int file_number, int B, int Z, const char * boconame,
 
      /* Save Point-Set on Disk */
     if (npnts > 0) {
-        char_33 PointSetName;
-        strcpy(PointSetName, PointSetTypeName[boco->ptset->type]);
-        if (cgi_write_ptset(boco->id, PointSetName, boco->ptset, index_dim,
+        HDF5storage_type = CG_CONTIGUOUS;
```

Comment:
Changing to contiguous could hurt performance for cases not doing partial I/O. Maybe just change it if the point list is NULL?

Reply:
I was thinking either that or if `ptset->type == PointList`? I feel like having it change based on `NULL` might be more confusing, where as a consistent "PointLists are contiguous, PointRanges are compact" is a bit clearer.

Reading on ["compact vs contiguous"](https://docs.hdfgroup.org/hdf5/develop/_l_b_dset_layout.html), I'm not sure what the definition of "very small data sets" is, but I can't imagine a `PointList` being any less than a few thousand entries. Comparatively, `PointRange` should at most be like 6 entries, so that's an obvious candidate for compact.

Reply:
64KiB raw data is the size limit for compact storage. Setting ptset->type == PointList will work.